tests/config.go

```diff
@@ -53,47 +39,21 @@ type KubeVirtTestsConfiguration struct {
 	StorageClassWindows string `json:"storageClassWindows"`
 }
 
-// Returns a new KubeVirtTestsConfiguration with default values
-func NewKubeVirtTestsConfiguration() *KubeVirtTestsConfiguration {
-	config := &KubeVirtTestsConfiguration{}
-
-	err := loadConfigFromFile(DefaultConfigFile, config)
-
-	if err != nil {
-		panic(fmt.Sprintf("Couldn't load default test suite configuration: %s\n", err))
-	}
-
-	return config
-}
-
-func loadConfig() *KubeVirtTestsConfiguration {
-	config := NewKubeVirtTestsConfiguration()
-
-	if ConfigFile != "" {
```

Comment:
do we care that we're losing the ability to set (configfile == "") and bypass loading a config?

Reply:
the config was already loaded from the default file at this place, see NewKubeVirtTestsConfiguration() a few lines above

Reply:
That all is still a little bit messed up, but definitely better now. :+1: